April 2020 update draft

ink7047 opened the pull request:
Hi Andre,

Sending this to you for your review/so you can add your update under Ruby Together news. Please note:

-Values like #s and names are missing from the Ruby Together news section I wasn’t able to get bin/create_update to work (or if there was another script for that I don’t have it/don’t remember how to run it )

-I will wait until I get the email on the 15th to add the budget if that works for you! Or if you’d like me to publish earlier please send me the April budget report. Ty!

-Let me know if I’m missing anything

indirect commented on app/views/news/2020-04-28-April-2020-monthly-update.html.md:
```diff
@@ -0,0 +1,64 @@
+* General & Administrative $
+* IT & Software $
+* Professional Fees	$
+* 123.5 hours of development work $
```
We're waiting on our bookkeepers for the final categorized budget, and they should have that by the 15th.

indirect commented on app/views/news/2020-04-28-April-2020-monthly-update.html.md:
```diff
@@ -0,0 +1,64 @@
+---
+summary: In March, we paid for 123.5 hours of developer work, retired the Bundler repository and GitHub organization, and completed many fixes and improvements for the RubyGems.org website.
```
Once we have the other sections written, update here with 1-2 sentences that include the total hours and maybe 1-2 things that we did.

ink7047 commented:
Thanks for these! I believed my revisions are in the commits I pushed today? Let me know if you can see them!

ink7047 commented:
Hi, I revised the post and added the budget data:

-In rubygems news: is it possible to create a link that jumps to a certain section of the page (e.g. in this case have the [merging] link jump to the "bundler news" section of the March 2020 update?)

-In budget & expenses: I removed the hours of development work because I didn't see that section in the report, please let me know where I can find it if you'd like me to add it back in.  

Thank you!

indirect commented:
@ink7047 okay, I think I figured out the answers to your questions! :)

>  is it possible to create a link that jumps to a certain section of the page

When you first asked, there was no way to link to a certain section, but that's a good idea! I adjusted the site to make it possible. Here's how to find out the section link name:

![CleanShot 2020-05-17 at 12 59 52@2x](https://user-images.githubusercontent.com/78/82158906-b4afc080-983f-11ea-8c9e-20605e3de5d9.png)

So in this case, the link name is `bundler.news`. To make a link directly to that section, you take the link to the whole page, and then add `#`, and then add the link name. So to link directly to the Bundler News section from March 2020, the link would be:

```
https://rubytogether.org/news/2020-04-02-March-2020-monthly-update#bundler.news
```

> I removed the hours of development work because I didn't see that section in the report, please let me know where I can find it

In the QuickBooks report, the line named "Program Expenditures" is how much we paid developers. To calculate hours, divide that by 150. So in this case, something like:

> 111.4 hours of development work $16,720.30

And then you can take the 111.4 number and put it up at the top in the summary.
